journal/original/11-chapter-8.adoc

```diff
@@ -1390,3 +1410,4 @@ which is sufficient in all temptations, to lead out of them, and to keep over th
 
 [.signed-section-signature]
 G+++.+++ F.
+//Not sure what I should do with all of this in the footnote.
```

Comment:
Yeah, sorry about this.  There's a little bug in the online editor that is causing it to look like the footnote goes on forever, but it actually doesn't. So, we missed some embedded documents here.  I'll try to fix this in the editor today, then you can go back to this file and it should be pretty clear the ones you missed.